tests/js/spec/views/organizationIntegrations/pluginDetailedView.spec.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import {Client} from 'app/api';
+import {mountWithTheme} from 'sentry-test/enzyme';
+import PluginDetailedView from 'app/views/organizationIntegrations/pluginDetailedView';
+import * as modal from 'app/actionCreators/modal';
+
+const mockResponse = mocks => {
+  mocks.forEach(([url, body]) => {
+    return Client.addMockResponse({
+      url,
+      body,
+    });
+  });
+};
+
+describe('PluginDetailedView', function() {
+  const org = TestStubs.Organization();
+  const routerContext = TestStubs.routerContext();
+  let wrapper;
+
+  beforeEach(() => {
+    Client.clearMockResponses();
+
+    mockResponse([
+      [
+        `/organizations/${org.slug}/plugins/configs/?plugins=pagerduty`,
+        [
+          {
+            status: 'unknown',
+            description: 'Send alerts to PagerDuty.',
+            isTestable: true,
+            isHidden: false,
```

Comment:
@NisanthanNanthakumar Can you change `isHidden` to `true`? A PR I am working on will cause this test to fail unless we change `isHidden` in this test: https://github.com/getsentry/sentry/pull/17165

Reply:
@scefali ok can do, but you need to merge your PR first.

Reply:
@NisanthanNanthakumar Nah, it will be fine to make that change before I merge my PR

Reply:
Or if this one gets merged first I can just update the test. We just have to make sure we both don't merge without rebasing or Sentry will break lol

Reply:
@NisanthanNanthakumar probably safer to just get this PR merged first in case I missed a case. I will fix any tests my PR breaks.